wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/MessageType.kt

```diff
@@ -59,6 +59,9 @@ data class MessageType(
   val requiredFields: List<Field>
     get() = fieldsAndOneOfFields.filter { it.isRequired }
 
+  val declaredFieldsAndOneOfFields: List<Field>
+    get() = declaredFields + oneOfs.flatMap { it.fields }
```

Comment:
Is it legal to use OneOf in an extension? If so, there's some additional work to be done here

Reply:
Could we move it in a swift modules as we've done for `isExtensible` ?

Reply:
That also mean that whatever you need to do, blank check

Reply:
sure

Reply:
done